doc/Command_Index_User.tex

```diff
@@ -154,66 +147,126 @@ \subsubsection{Including scores}
+  \#3 & 0 & Distance will not scale when staff size is changed.\\
+  & 1 & Distance will scale when staff size is changed.
 \end{argtable}
 
-Examples:\par\medskip
+\macroname{\textbackslash grescaledim}{\{\#1\}\{\#2\}}{gregoriotex-spaces.tex}
+Macro to turn on or off scaling with the staff size for a particular distance.
+
+\begin{argtable}
+  \#1 & string & The name of the distance for which scaling it to changed.  See \nameref{distances} below.\\
+  \#2 & \texttt{yes}/\texttt{true}/\texttt{on} & Scale the distance when changing the size of the staff.\\
+  & \textit{other} & Do not scale the distance when changing the size of the staff.
+\end{argtable}
+
+\macroname{\textbackslash greloadspaceconf}{\{\#1\}}{gregoriotex-spaces.tex}
+Macro to load a space configuration file.  Space configuration file names have the format \verb=gsp-identifier.tex= and must be in the same directory as your project or in your texmf directory.  This function if \texttt{gre} prefixed to highlight that it only loads distances for Gregorio\TeX.
+
+\begin{argtable}
+  \#1 & string & The identifier of the space configuration file.
+\end{argtable}
+
+Example:\par\medskip
 \begin{latexcode}
-  \grechangestyle{translation}{\it\bf} % Works for both PlainTeX and LaTeX this would make the translations bold and italic
-  \grechangestyle{abovelinetext}{\begin{small}\begin{italic}}[\end{italic}\end{small}] % LaTeX only, this would make the above lines text small and italic
-  \grechangestyle{initial}{\fontsize{36}{36}\selectfont} % This would make the initial print in 36pt font.
+  % loads gsp-default.tex, the default configuration file
+  \greloadspaceconf{default}
+  % loads a custom configuration called gsp-myspaces.tex
+  \greloadspaceconf{myspaces}
 \end{latexcode}
 
-Each element will be typeset within an isolated group to prevent styling commands from leaking from one element to the next.  As a result, if a styling command has an ``on-switch'' but no ``off-switch'' (like \verb=\it= or \verb=\bf= in the first example above) it is not necessary to encapsulate them within a \verb=\begingroup= and \verb=\endgroup=.  As a result, the third argument is only necessary for styling commands which come in pairs (like the environments in the second example).
 
-\subsubsection{Alternate glyph drawings}
+\subsubsection{Staff Lines}
+
+
+\macroname{\textbackslash gresetlinecolor}{\{\#1\}}{gregoriotex.sty \textup{and} gregoriotex.tex}
+Macro for changing the color of the staff lines.   The two most common colors you're going to want to use are \texttt{gregoriocolor} (see \nameref{colors}) and \texttt{black} (the default).
+
+\begin{argtable}
+  \#1 & \textit{color name} & The color of the staff lines
+\end{argtable}
+
+\macroname{\textbackslash gresetlines}{\{\#1\}}{gregoriotex-main.tex}
+Macro for setting whether the staff lines should be printed or not.
```

Comment:
"printed" and "shown" are used interchangeably in this section of the documentation, but I think a consistent word should be used.  I would actually use the word "rendered" (which applies to both screen and print media) or "engraved" (since that is the purported purpose of the application), but that's just my opinion.